### DISCUSSION TITLE
Adding query params to URL makes the page reload without stopping

### DISCUSSION THREAD

**Discussion by choobinsan** [vercel/community#526] [Help]
Hi there.
We have a NEXT JS app ans we are using ISR with _getStaticProps_

And we use [[slug]].tsx for dynamic routing.

And we have a couple of places where we use:
`return {
            notFound: true,
            revalidate: 5,
        };`

The issue we are currently facing is that, as soon as a query parameter is added to the url, the page starts reloading itself without ever stopping.
An example would be:
https://www.xyz.com which works fine
https://www.xyz.com?a=123 which makes the page reload reload without stopping

Any help would be appreciated.
Thanks in advance

**Comment by leerob** [comment]
Hey! Could you create a minimal reproduction of this issue? This doesn’t sound like an issue with ISR on the surface level.

**Reply by choobinsan** [reply]
@leerob Hi, thanks for your reply. You mean code snippets? I don't think I can do that. 
But we have quite a simple slug file and all it does in the getStaticProps is fetching data from a CMS. 
Currently we have fixed the issue via change to getServerSideProps

**Reply by gogocharli** [reply]
Hello @devarvind, could you [open a case](https://vercel.com/support/request) around this issue? We would like to investigate it further.

**Reply by choobinsan** [reply]
@gogocharli Yes of course! I'll do it right away
